style.css

```diff
@@ -502,3 +500,269 @@ textarea::placeholder {
   font-family: "Poppins", sans-serif;
   color: #b3bac5;
 }
+
+.lang-list {
+  align-items: center;
+  padding: 24px;
+  gap: 12px;
+  background: #f7f7f9;
+  border-radius: 8px;
+}
+
+@media (min-width: 768px) {
+  .nav {
+    margin-right: 7.75rem;
+  }
+
+  .nav ul {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    list-style: none;
+    padding: 0 1rem 0 1rem;
+    width: 16.188rem;
+    margin-right: 8.75rem;
+  }
+
+  .nav li {
+    font-family: "poppins", sans-serif;
+    color: #334563;
+    font-weight: 500;
+    font-size: 15px;
+    padding: 0 0.75rem;
+  }
+
+  .hamburger {
+    /* color: red; */
+    visibility: hidden;
+  }
+
+  .skill-section ul {
+    /* display: flex;  1 */
+    padding-left: 20px;
+  }
+
+  /* .skill-dropdown-content li {
+    height: 66px;
+    background-color: #f7f7f9;
+    width: 100%;
+    padding: 8px 16px;
+    display: flex;
+    border-radius: 8px;
+  } */
+
+  .skill-section ul li {
+    /* flex-direction: column;2 */
+    height: 100px;
+
+    /* margin-right: 12px;3 */
+  }
+
+  .skill-dropdown-content {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    padding: 12px 32px;
+    gap: 12px;
+  }
+
+  /* .skill-dropdown-content li {
+    height: 120px;
+    width: 122px;
+    background-color: #f7f7f9;
+    padding: 8px 0;
+    display: flex;
+    flex-direction: column;
+    border-radius: 8px;
+    text-align: center;
+  } */
+
+  .nav li:enabled {
+    color: #6070ff;
+    background: #fff;
+  }
+
+  .nav li:hover {
+    color: #fff;
+    background: #6070ff;
+  }
+
+  .nav li:active {
+    color: #fff;
+    background: #2230d2;
+  }
+
+  .nav li:disabled {
+    color: #5e6c84;
+    background: #fff;
+  }
+
+  .content {
+    background-image: url(./images/HeaderD.png);
+    background-position: right;
+    background-size: contain;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 650px;
+  }
```

Comment:
- [ ] As one of the requirement in the project is to parse a figma design as close to as possible, let's fix the headline section to have a height of 100vh so that this section covers the viewport height when we visit out portfolio page, currently it is looking like this 

![image](https://user-images.githubusercontent.com/97521557/208237716-f66bbabb-183b-4dad-935b-51eb7cbff9bd.png)

how it should be according to the template 
![image](https://user-images.githubusercontent.com/97521557/208237814-a2bcd7eb-689e-4bbb-b64c-8a3197198db8.png)

- [ ] Please also give some padding top to your header or nav section as currently it is to close to the top please see 
![image](https://user-images.githubusercontent.com/97521557/208237869-136a6cea-9ff7-483c-a6db-384bccbdd001.png)

- [ ] Please also make sure the content of our headline section is properly aligned as it is in the figma template currently your design is looking like 
![image](https://user-images.githubusercontent.com/97521557/208237947-683c8b39-b0c7-443b-8f7a-c51c3a863338.png)

How it should be according to the template, notice how the content is perfectly aligned at the start
![image](https://user-images.githubusercontent.com/97521557/208237974-362fbedf-ab3e-458f-a890-ebb76ef2f989.png)